deploy/docs/SideBySidePrometheus.md

```diff
@@ -1,6 +1,10 @@
 # How to install our Prometheus side by side with your existing Prometheus
 
-When installing our Helm Chart it is possible to have more than one Prometheus server running in the same cluster. However, do note that you cannot have more than one Prometheus Operator running in the same cluster. This document will take you through the steps to set up Sumo Logic collection when you have an existing Prometheus Operator you wish to keep intact.
+When installing our Helm Chart it is possible to have more than one Prometheus server running in the same cluster.
+However, do note that you must either limit the scope of the interaction of the Prometheus Operator
+(by setting option `kube-prometheus-stack.prometheusOperator.namespaces.additional`)
+or disable Prometheus Operator in Sumo Logic's Kubernetes collection (existing Prometheus Operator must have `namespace` in which Sumo Logic's Kubernetes collection in the scope of its interaction).
```

Comment:
> must have `namespace` in which Sumo Logic's Kubernetes collection in the scope of its interaction

I think the explanation is too complex

Reply:
Prometheus Operator must have correctly set namespace parameters to be able to capture configuration in Prometheus resource from the namespace in which collection is installed...
Following parameters should be taken into account:
- namespaces: https://github.com/prometheus-operator/prometheus-operator/blob/a45136e4f591f5fff6748b71fe615477c2af6e87/cmd/operator/main.go#L163
- deny-namespaces :https://github.com/prometheus-operator/prometheus-operator/blob/a45136e4f591f5fff6748b71fe615477c2af6e87/cmd/operator/main.go#L164
- prometheus-instance-namespaces: https://github.com/prometheus-operator/prometheus-operator/blob/a45136e4f591f5fff6748b71fe615477c2af6e87/cmd/operator/main.go#L165

and best documentation that I've found for this parameters is in the code...

Reply:
ok, I've found the docs that I can reference in our doc https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/operator.md

Reply:
@sumo-drosiek please check if additional information helps